src/assets/scripts/client/UiController.js

```diff
@@ -309,13 +309,12 @@ class UiController {
         const parsedSelectedOption = !_isNaN(parseFloat(selectedOption))
             ? parseFloat(selectedOption)
             : selectedOption;
-        let optionSelectTempalate = `<option value="${optionData.value}">${optionData.displayLabel}</option>`;
 
         if (optionData.value === parsedSelectedOption) {
-            optionSelectTempalate = `<option value="${optionData.value}" selected>${optionData.displayLabel}</option>`;
```

Comment:
hmm, why the change to a return here?

Reply:
So we don't do the string templating twice.